WPFHospitalEditor/AlternativeRelocationAppointments.xaml.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Windows;
+using WPFHospitalEditor.Controller;
+using WPFHospitalEditor.Controller.Interface;
+using WPFHospitalEditor.MapObjectModel;
+
+namespace WPFHospitalEditor
+{
+    /// <summary>
+    /// Interaction logic for AlternativeRelocationAppointments.xaml
+    /// </summary>
+    public partial class AlternativeRelocationAppointments : Window
+    {
+        public IMapObjectController mapObjectController = new MapObjectController();
+        public IRoomServerController roomServerController = new RoomServerController();
+
+        MapObject mapObject = null;
```

Comment:
You don't need field which is initialized and used in only one method (ShowRoomOnMap()).

Reply:
Fixed, it is in method now.